.github/stale.yml

```diff
@@ -0,0 +1,35 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 90
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - "Priority: Critical"
+  - "Priority: High"
+  - "Priority: Medium"
+  - "Priority: Low"
+  - "PR: Major"
+  - "PR: Minor"
+  - "PR: Patch"
+  - "Type: Bug"
+  - "Type: Security"
+  - "Type: Nice 1st Contribution"
```

Comment:
maybe add in `Status: WIP` as well?

Reply:
and maybe `Status: On Hold`?

Reply:
Yeah

Reply:
Should be fine now.

Reply:
:+1:

Reply:
Lets try it out and we can always re-open ones it closes by mistake